packages/account-abstraction-auth/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@safe-global/account-abstraction-auth",
```

Comment:
Should we rename the package to `auth-provider`? (same for the folder name)
In that case we can have something like:
- account-abstraction-sdk (monorepo)
  - account-abstraction-sdk (maybe we can find a better name in the future)
  - relay-provider
  - auth-provider

Reply:
thing is auth-provider is generic as well in the @safe-global context

Reply:
so users are going to use @safe-global/auth-provider in their projects